legal/satzung.md

```diff
@@ -0,0 +1,122 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung ebendieser Software, und
+    3. $(ZWECK)
+
+
+## Selbstlosigkeit
+
+1. Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins dürfen nur für seine satzungsmäßigen Zwecke verwendet werden.
+    Die Mitglieder dürfen keine Gewinnanteile und in ihrer Eigenschaft als Mitglieder auch keine sonstigen Zuwendungen aus Mitteln des Vereins erhalten.
+3. Es darf keine Person durch Ausgaben, die dem Zweck des Vereins fremd sind, oder durch unverhältnismäßig hohe Vergünstigungen begünstigt werden.
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwärt*in.
```

Comment:
Other options to have this gender-neutral?

Reply:
Müsste man nicht jeweils der/dem schreiben? Oder geht auch der*m?

Reply:
> Müsste man nicht jeweils der/dem schreiben? Oder geht auch der*m?

Eigentlich vermutlich "der/dem", aber ich halte generisches Femininum für okay, solange vom Substantiv her klar ist, dass Männer mitgemeint sind.